lib/CrEOF/Spatial/PHP/Types/AbstractGeometry.php

```diff
@@ -65,6 +65,14 @@ public function toJson()
         return json_encode($json);
     }
 
+    /**
+     * @return array
+     */
+    public function jsonSerialize()
+    {
+        return $this->toArray();
```

Comment:
Also type should be in the json data to make compatible to GeoJSON spec. Better implementation:

```php
public function jsonSerialize()
{
    return ['type' => $this-getType(), 'coordinates' => $this->toArray()];
}
```